src/cpu/aarch64/matmul/acl_matmul.hpp

```diff
@@ -85,13 +85,18 @@ struct acl_matmul_t : public primitive_t {
                     = utils::everyone_is(data_type::f16, src_md()->data_type,
                               weights_md()->data_type, dst_md()->data_type)
                     && platform::has_data_type_support(data_type::f16);
+            const bool is_bf16_ok
+                    = utils::everyone_is(data_type::bf16, src_md()->data_type,
+                              weights_md()->data_type, dst_md()->data_type)
+                    && platform::has_data_type_support(data_type::bf16);
 
             // we need to save this state as it can change inside set_default_formats()
             weights_format_kind_ = weights_md_.format_kind;
 
             VDISPATCH_MATMUL(
                     is_dense_format_kind(), VERBOSE_UNSUPPORTED_SPARSE_CFG);
-            VDISPATCH_MATMUL(utils::one_of(true, is_fp32_ok, is_fp16_ok),
+            VDISPATCH_MATMUL(
+                    utils::one_of(true, is_fp32_ok, is_fp16_ok, is_bf16_ok),
```

Comment:
Hi @renato-arantes , thanks for the changes. Is bf16 matmul covered by validation on your side? I see there are no changes to the benchdnn inputs.

Reply:
Thanks!